examples/step-10/doc/results.dox

```diff
@@ -217,3 +226,97 @@ convergence. (This effect is also discussed in detail in the following
+no longer decreases with mesh refinement as anticipated. This is because both
+within this code and within the many computations that happen within deal.II
+itself, each operation incurs an error on the order of $10^{-16}$; adding
+such errors many times over then results in an error that may be on the
+order of $10^{-14}$, which will dominate the discretization error after
+a number of refinement steps and consequently destroy the convergence rate.
+
+The question is whether one can do anything about this. One thought is to
+use a higher-precision data type. For example, one could think of declaring
+both the `area` and `perimeter` variables in `compute_pi_by_area()` and
+`compute_pi_by_perimeter()` with data type `long double`. `long double`
+is a data type that is not well specified in the C++ standard but at least
+on Intel processors has around 19, instead of around 16, digits of accuracy.
+If we were to do that, we would get results that differ from the ones shown
+above. However, maybe counter-intuitively, they are not uniformly better.
+For example, when computing $\pi$ by the area, at the time of writing
+these sentences we get these values with `double` precision for degree 4:
+@code
+    5 3.1415871927401144 5.4608e-06    -
+   20 3.1415926314742491 2.2116e-08 7.95
+   80 3.1415926535026268 8.7166e-11 7.99
+  320 3.1415926535894005 3.9257e-13 7.79
+ 1280 3.1415926535899774 1.8430e-13 1.09
+ 5120 3.1415926535897669 2.6201e-14 2.81
+@endcode
+On the other hand, the results are as follows when using `long double`:
+@code
+ cells      eval.pi            error
+    5 3.1415871927401136 5.4608e-06    -
+   20 3.1415926314742446 2.2116e-08 7.95
+   80 3.1415926535026215 8.7172e-11 7.99
+  320 3.1415926535894516 3.4157e-13 8.00
+ 1280 3.1415926535897918 1.5339e-15 7.80
+ 5120 3.1415926535897927 5.2649e-16 1.54
+@endcode
+Indeed, here we get results that are approximately 50 times as accurate.
+On the other hand, when computing $\pi$ by the perimeter, we get this with
+`double` precision:
+@code
+    5 3.1415921029432572 5.5065e-07    -
+   20 3.1415926513737582 2.2160e-09 7.96
+   80 3.1415926535810699 8.7232e-12 7.99
+  320 3.1415926535897576 3.5527e-14 7.94
+ 1280 3.1415926535897896 3.5527e-15 3.32
+ 5120 3.1415926535897940 8.8818e-16 2.00
+@endcode
+Whereas we get the following with `long double`:
+@code
+    5 3.1415921029432572 5.5065e-07     -
+   20 3.1415926513737595 2.2160e-09  7.96
+   80 3.1415926535810703 8.7230e-12  7.99
+  320 3.1415926535897576 3.5705e-14  7.93
+ 1280 3.1415926535897918 1.3785e-15  4.70
+ 5120 3.1415926535897944 1.3798e-15 -0.00
+@endcode
+Here, using `double` precision is more accurate by about a factor of
+two. (Of course, in all cases, we have computed $\pi$ with more
+accuracy than any engineer would ever want to know.)
+
+What explains this unpredictability? In general, round-off errors can
+be thought of as random that add up in ways that are not worth thinking
```

Comment:
>as random

Perhaps this is incomplete -- maybe `as random errors`? Or `as random, and add up in ways...`

Reply:
Yes, the latter is what I meant.